app/assets/scripts/views/dashboard.js

```diff
@@ -126,23 +140,41 @@ var Dashboard = React.createClass({
     );
   },
 
+  renderStats: function () {
+    let { fetched, data } = this.props.userTasks;
+
+    if (!fetched) {
+      return null;
+    }
+
+    return (
+      <div className='tasks-stats'>
+        <h2 className='tasks-stats__title'>Tasks</h2>
+        <ul className='stats-list'>
+          <li className='stats-list__item'><strong>{data.stats.activeTasks}</strong>active</li>
+          <li className='stats-list__item'><strong>{data.stats.completedTasks}</strong>completed</li>
```

Comment:
This was my bad. I made a change to ensure consistent naming but forgot to push :S
Here it is: https://github.com/hotosm/imagery-requests-api/pull/16